qutebrowser/commands/userscripts.py

```diff
@@ -144,18 +180,19 @@ def _cleanup(self):
                         fn, e))
         self._filepath = None
         self._proc = None
-        self._env = None
+        self._env = {}
+        self._text_stored = False
+        self._html_stored = False
 
-    def run(self, cmd, *args, env=None, verbose=False):
-        """Run the userscript given.
+    def prepare_run(self, *args, **kwargs):
+        """Prepare ruinning the userscript given.
```

Comment:
That's what I get for using vim (or well, emacs with vim keybindings) I guess :laughing: 

(But what if I want to ruin the userscript?)